### ISSUE TITLE
[HEADINGS]: GIBCT EYB - Additional resources headings SHOULD be H2s on all three profile types

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#14419]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to hear headings announced in the proper nesting order. It's potentially confusing when I hear the H1 read aloud, then skip to an H3, then back to a series of H2's. Users have reported this is confusing, and they wonder if they've missed important content.

## Details

The VETTEC Additional Resources heading is an H2, and the higher education and OTJ Additional Resources headings are H3s. They should all be H2s. Screenshots attached below.

## Acceptance Criteria

- [ ] All Additional Resources headings are H2s
- [ ] Visual designs do not change from current
- [ ] Screen readers announce the H2s in the proper nesting order

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Environment

* https://staging.va.gov/gi-bill-comparison-tool/profile/11006124
* https://staging.va.gov/gi-bill-comparison-tool/profile/10000008
* https://staging.va.gov/gi-bill-comparison-tool/profile/3V000142/CERTIFIED%20INFORMATION%20SYSTEMS%20SECURITY

## WCAG or Vendor Guidance (optional)

* [Headings and Labels: Understanding SC 2.4.6](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html)

## Screenshots or Trace Logs

![Screen Shot 2020-10-05 at 3 51 58 PM](https://user-images.githubusercontent.com/934879/95130895-d6441280-0722-11eb-9578-a7352df7c625.png)

---
![Screen Shot 2020-10-05 at 3 52 06 PM](https://user-images.githubusercontent.com/934879/95130911-dd6b2080-0722-11eb-838e-fc44cb9c8adb.png)

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/14522

**Comment by sonntag-adam** [comment]
![Screen Shot 2020-10-09 at 2.30.38 PM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/064fd660-c5fe-4b0d-8bba-5f9da6b903e2)
##
![Screen Shot 2020-10-09 at 2.31.39 PM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/c664babc-1e4f-4d4c-ba29-dee3ee9374cc)
##
![Screen Shot 2020-10-09 at 2.32.03 PM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/607cd36e-916d-4905-b021-19798dfb6252)

**Comment by 1Copenut** [comment]
👍 @sonntag-adam I'll move this one to close.
